SQL/0000-00-00-schema.sql

```diff
@@ -1552,20 +1551,20 @@ CREATE TABLE `parameter_type` (
 
 
 INSERT INTO `parameter_type` VALUES
-  (2,'Geometric_distortion','text',NULL,NULL,NULL,NULL,'parameter_file',NULL,NULL,0,0),
-  (3,'Intensity_artifact','text',NULL,NULL,NULL,NULL,'parameter_file',NULL,NULL,0,0),
-  (4,'Movement_artifacts_within_scan','text',NULL,NULL,NULL,NULL,'parameter_file',NULL,NULL,0,0),
-  (5,'Movement_artifacts_between_packets','text',NULL,NULL,NULL,NULL,'parameter_file',NULL,NULL,0,0),
-  (6,'Coverage','text',NULL,NULL,NULL,NULL,'parameter_file',NULL,NULL,0,0),
-  (7,'md5hash','varchar(255)','md5hash magically created by NeuroDB::File',NULL,NULL,'parameter_file.Value','parameter_file',NULL,'quat_table_1',1,0),
-  (8,'Color_Artifact','text',NULL,NULL,NULL,NULL,'parameter_file',NULL,NULL,0,0),
-  (9,'Entropy','text',NULL,NULL,NULL,NULL,'parameter_file',NULL,NULL,0,0);
-
-
-INSERT INTO parameter_type (Name, Type, Description, RangeMin, RangeMax, SourceField, SourceFrom, CurrentGUITable, Queryable, SourceCondition) VALUES
```

Comment:
It is better to keep the values list.
(Name, Type, Description, RangeMin, RangeMax, SourceField, SourceFrom, CurrentGUITable, Queryable, SourceCondition)

Reply:
I am just removing one column from the entire table

Reply:
So?
@kongtiaowang  comment is valid and can prevent error in the future. You can also add the attributes list on line 1553 (copy&paste from line 2134 or RBdata.sql)  and also just not include the attributes that are always null. 

On line 1564, should have back-tick around table name.

Reply:
I'm not sure what you guys are talking about.. it looks like he is keeping the explicit column list, he's just removing CurrentGUITable from it since the column was dropped from the table?

Reply:
@PapillonMcGill I'm not removing the list... not sure what the issue is here

Reply:
@ridz1208  see comments for line 1553

Reply:
so just to be clear there is no issue here ? the issue is only on line 1553

Reply:
both line are ok now!